src/backend/distributed/connection/remote_commands.c

```diff
@@ -52,7 +54,7 @@ ForgetResults(MultiConnection *connection)
 	while (true)
 	{
 		PGresult *result = NULL;
-		result = PQgetResult(connection->pgConn);
+		result = GetRemoteCommandResult(connection, false);
```

Comment:
It's worth following our convention of using a raiseInterrupts variable as an argument such that the reader knows what the parameter does without looking at the function.